processor/src/main/java/org/mapstruct/ap/model/common/Type.java

```diff
@@ -619,7 +617,9 @@ private int distanceTo(TypeMirror base, TypeMirror targetType) {
     }
 
     /**
-     * Whether this type can access the given method declared on the given type.
+     * @param type the type declaring the method
```

Comment:
I don't think the param tags add much value here. Let's don't go overboard just to make the JavaDoc tool happy ;) The original form should be fine (and actually I intentionally use that style if @param/@return don't add any value.